CodeStarterCamp_Week1/main.swift

```diff
@@ -1,27 +1,41 @@
 //
-//  main.swift
+//  STEP3.swift
 //  CodeStarterCamp_Week1
 //
-//  Created by yagom.
-//  Copyright © yagom academy. All rights reserved.
+//  Created by Jay Shin on 2/12/24.
 //
 
 import Foundation
 
-func drawIceCreamBody(bodyWidth: Int, bodyHeight: Int) {
-    for _ in 1...bodyHeight {
-        for _ in 1...bodyWidth {
-            print("*", terminator: "")
-        }
-        print("")
+drawCookie(cookieLength: 10, cookieBase: "***", cookieTopping: " ", stickLength: 4)
+drawCookie(cookieLength: 12, cookieBase: "***", cookieTopping: "&", stickLength: 4)
+drawCookie(cookieLength: 12, cookieBase: "***", cookieTopping: "#", stickLength: 6)
+drawCookie(cookieLength: 6, cookieBase: "|0|", cookieTopping: " ", stickLength: 4)
+
+func drawCookie(cookieLength: Int, cookieBase: String, cookieTopping: String, stickLength: Int) {
```

Comment:
```suggestion
func drawCookie(length: Int, base: String, topping: String, stickLength: Int) {
```
`drawCookie`라는 이름만 보아도 이 함수는 쿠키를 그릴 것이라고 생각할 수 있을 정도로 직관적인 네이밍입니다. 👍

그렇기 때문에 내부 매개변수 이름에 굳이 `cookie`라는 단어를 반복하지 않아도 이해할 수 있을 것 같습니다. '쿠키에 웬 길이?'라고 생각할 수 있지만, 이는 앞선 코멘트처럼 빼빼로 형태의 막대과자임을 알 수 있게 네이밍이 변경된다면 직관적으로 이해할 수 있을 것 같습니다.

Reply:
매개변수명의 이름을 좀 더 가독성있게 축소했습니다.
수정하는 과정에서 매개변수가 어떻게 함수간에 이어져 있는지 정리가 되었습니다:)